### ISSUE TITLE
 Wrong results after Tensorflow 2x model conversion #70 

### ISSUE THREAD

**Issue by sergiocasaspastor** [tlatkowski/inpainting-gmcnn-keras#18]
Hi,

I hope you still give support for this repository. I tried converting this repository model to Tensorflow 2x step by step (also following the original repository by shepnerd). I think everything was fine, but when training and testing the model the results were not very satisfactory. So, maybe there is a problem with my conversion that I can't find it. I tried to train the model on a small subset of OpenImages V6. The principal parameters that I used (sorry, because names can be slightly different than yours):

--img_size 256x256x3
--batch_size 4
--learning_rate 1e-4
--gaussian_steps 7
--gaussian_kernel_size 32
--gaussian_kernel_std 20.0
--reconstruction_loss_weight 1.2
--adversarial_loss_weight 0.001
--gradient_penalty_loss_weight 10
--id_mrf_loss_weight' 0.03
--nn_stretch_sigma 0.5
--id_mrf_style_weight 1.0
--id_mrf_content_weight 1.0

I pretrained the model with only confidence reconstruction loss for the recommended steps, and results for this phase seem fine.

![imagen](https://user-images.githubusercontent.com/39574343/111080635-a08c2c80-84ff-11eb-80d2-77d97e9030c3.png)
![imagen](https://
![imagen](https://user-images.githubusercontent.com/39574343/111080638-a7b33a80-84ff-11eb-8d54-f42ed9aa29c6.png)
user-images.githubusercontent.com/39574343/111080637-a3871d00-84ff-11eb-90cb-c6de09d587f6.png)

However, in the training phase, after some steps, the results do not seem to converge and eliminate the mask.

![imagen](https://user-images.githubusercontent.com/39574343/111080653-bdc0fb00-84ff-11eb-9708-187f363a2dbd.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080656-c1ed1880-84ff-11eb-891d-17808c2f56f9.png)
![imagen](https://user-images.githubusercontent.com/39574343/111080668-d204f800-84ff-11eb-9665-a76e40ebde28.png)

I would like to know if you have any idea of what could be happening, or you have experienced any similar issue. I reviewed the code, networks, and losses many times, but could not find any solution.

Thank you very much.
